vnext/Microsoft.ReactNative/XamlView.h

```diff
@@ -43,7 +48,7 @@ inline int64_t GetTag(winrt::IPropertyValue value) {
 
 inline winrt::IPropertyValue GetTagAsPropertyValue(XamlView view) {
   assert(view);
-  return view.GetValue(xaml::FrameworkElement::TagProperty()).try_as<winrt::IPropertyValue>();
+  return view.GetValue(ReactTagProperty()).try_as<winrt::IPropertyValue>();
```

Comment:
we'll want to do a paired change to react-native-xaml since it plays shenanigans with the Tag (gets a tag on one element and sets it on another to trick React into thinking it's the same thing)

Reply:
does that fix the mapping in the shadow nodes?

Reply:
can you clarify? the shadow nodes are inaccessible to the view manager since it's an implementation detail not exposed through the winrt surface

Reply:
Right, so if you're just modifying the tag via the XAML property in RNX (presumably because you're moving things around), is the shadow tree still correct?